services/121-service/src/program-financial-service-provider-configurations/program-financial-service-provider-configurations.repository.ts

```diff
@@ -19,44 +23,95 @@ export class ProgramFinancialServiceProviderConfigurationRepository extends Repo
   public async findByProgramIdAndFinancialServiceProviderName(
     programId: number,
     financialServiceProviderName: FinancialServiceProviderName,
```

Comment:
Relations can be replaced with  a boolean includeProperties and make the params deconstucted object